test/unittests/util/test_time.py

```diff
@@ -0,0 +1,76 @@
+from datetime import datetime
+from dateutil.tz import tzfile, tzlocal, gettz
+from unittest import TestCase, mock
+
+from mycroft.util.time import (default_timezone, now_local, now_utc, to_utc,
+                               to_local, to_system)
+
+test_config = {
+    'location': {
+        'timezone': {
+            'code': 'America/Chicago',
+            'name': 'Central Standard Time',
+            'dstOffset': 3600000,
+            'offset': -21600000
```

Comment:
replace magic numbers with named constants

Reply:
Good idea, I never think about adding todos for these things because my editor always warns me that methods/functions are too complex :)

Reply:
I'm sceptical, this mirrors how it looks in the default config. Feels like it's better to keep it as close to the source as possible.

Reply:
I disagree.  At least put a comment as to what the numbers represent so someone doesn't have to come along later to do the math in their heads.

Reply:
Comments I agree is a good thing, added those.